### PR TITLE
Feedback meeting

### PR DESCRIPTION
**Pull request by adibalwani**

### CODE REVIEW

File: src/Common/Common.Api/Admin/Dataset.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+using System;
+
+namespace Microsoft.PowerBI.Common.Api.Encryption
```
Comment (CodeCyclone) [inline src/Common/Common.Api/Admin/Dataset.cs]: Namespace should match directory structure

File: src/Common/Common.Api/Admin/EncryptionKey.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+ 
+using System;
+
+namespace Microsoft.PowerBI.Common.Api.Encryption
```
Comment (CodeCyclone) [inline src/Common/Common.Api/Admin/EncryptionKey.cs]: Similar comment to the other file. You could create a directory structure for Encryption too but right now it should be Admin based on directory structure

File: src/Common/Common.Api/Admin/Dataset.cs (hunk 1 above)
Comment (adibalwani) [reply src/Common/Common.Api/Admin/Dataset.cs]: Moved to Encryption directory

File: src/Common/Common.Api/Admin/EncryptionKey.cs (hunk 2 above)
Comment (adibalwani) [reply src/Common/Common.Api/Admin/EncryptionKey.cs]: Moved to Encryption directory

**Pull request merged**